### ISSUE TITLE
Name ambiguity

### ISSUE THREAD

**Issue by romgrk** [smolck/nvim-todoist.lua#1]
Hey :)

Just wanted to highlight that one of my projects is already using the name [todoist.nvim](https://github.com/romgrk/todoist.nvim). If you want to publicize this project, might I ask that you pick a different name to ensure that both are differentiable?

Thanks

**Comment by smolck** [comment]
@romgrk Yeah for sure! Would `nvim-todoist.lua` work, or did you have something else in mind that would be less ambiguous?

**Comment by romgrk** [comment]
That is perfect, I don't have strong opinions, anything that makes the difference clear is cool.

And btw, I don't know if you're coding this for fun or for utility, but if it's for the utility I'd be happy to have people working with me on the plugin I started :)

**Comment by smolck** [comment]
Great! Changed :) I apologize for not doing that from the start!

I’ll close this issue once I update the Lua module name.

> And btw, I don't know if you're coding this for fun or for utility, but if it's for the utility I'd be happy to have people working with me on the plugin I started :)

todoist.nvim is really cool, but because of https://github.com/romgrk/todoist.nvim/issues/16 I was unable to use it, so I decided I’d start this project. Before I did that I looked around todoist.nvim’s code to see if I could fix that issue, but I wasn’t really sure why the issue was happening, so I gave up and started working on this. That was my original motivation with this project, but also doing it in pure Lua has been and is a good learning experience regarding writing Lua plugins.
